recipes/rmm/all/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.15)
+project(RMM CXX)
+
+if(NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/RMM_RAPIDS.cmake)
+  file(DOWNLOAD https://raw.githubusercontent.com/rapidsai/rapids-cmake/branch-23.04/cmake/write_version_file.cmake
+       ${CMAKE_CURRENT_BINARY_DIR}/write_version_file.cmake)
+endif()
+include(${CMAKE_CURRENT_BINARY_DIR}/write_version_file.cmake)
```

Comment:
Why is this necessary? It is a header-only library, not being built. It seems this whole file is not used?

Reply:
It's not. Sorry about that. Forgot to delete this after some initial experiments.